docker/secondary-analysis-python/tools/file_utils/file_utils/utils.py

```diff
@@ -0,0 +1,28 @@
+import requests
+
+
+def get_file_by_uuid(file_id):
```

Comment:
I like to include `verbose` flags in the definition of function like these in case one might want to interact with stdout or silence printing for some reason later. 

How do you feel about adding that? 

Here it would just be: 
```
def get_file_by_uuid(file_id, verbose=True):
    url = 'https://dss.dev.data.humancellatlas.org/v1/files/{}?replica=gcp'.format(file_id)
    response = requests.get(url)
    if verbose:
        print('GET {0}'.format(url))
        print('{0}'.format(response.status_code))
        print('{0}'.format(response.text))
    return response.json()
```

which would have the exact same invocation, but adds a bit more reusability.